src/composables/useWindowingConfigInitializer.ts

```diff
@@ -96,94 +27,52 @@ export function useWindowingConfigInitializer(
     return undefined;
   });
 
-  watchImmediate(windowConfig, (config) => {
-    const image = imageData.value;
-    const imageIdVal = unref(imageID);
-    const viewIdVal = unref(viewID);
-    if (config || !image || !imageIdVal) return;
-
-    const [min, max] = image.getPointData().getScalars().getRange();
-    store.updateConfig(viewIdVal, imageIdVal, { min, max });
-    store.resetWindowLevel(viewIdVal, imageIdVal);
-  });
-
-  watchImmediate(scalarRange, (range) => {
-    const imageIdVal = unref(imageID);
-    const viewIdVal = unref(viewID);
-    if (!range || !imageIdVal || !viewIdVal) return;
-    store.updateConfig(viewIdVal, imageIdVal, { min: range[0], max: range[1] });
-  });
-
-  function updateConfigFromAutoRangeValues() {
+  function resetWidthLevel() {
     const imageIdVal = unref(imageID);
     const viewIdVal = unref(viewID);
     if (imageIdVal == null) {
       return;
     }
 
-    if (autoRange.value in autoRangeValues.value) {
-      const [min, max] = autoRangeValues.value[autoRange.value];
-      store.updateConfig(viewIdVal, imageIdVal, {
-        min,
-        max,
-      });
-    }
+    store.updateConfig(viewIdVal, imageIdVal, {
+      auto: WL_AUTO_DEFAULT,
+    });
 
     const firstTagVal = unref(firstTag);
     if (firstTagVal?.width) {
       store.updateConfig(viewIdVal, imageIdVal, {
-        preset: {
-          width: firstTagVal.width,
-          level: firstTagVal.level,
-        },
+        width: firstTagVal.width,
+        level: firstTagVal.level,
       });
     }
 
-    const forcedWL = store.runtimeConfigWindowLevel;
-    if (forcedWL) {
+    const jsonWidthLevel = store.runtimeConfigWindowLevel;
+    if (jsonWidthLevel) {
```

Comment:
nit: just name this as `widthLevel`. Prefixing with `json` makes it seem like the type is a string containing json, when it's just an object.

Reply:
Fixed.